source/codegen/templates/service_helpers.mako

```diff
@@ -65,8 +65,9 @@ ${set_response_values(output_parameters)}\
 %>\
 ${initialize_input_params(function_name, parameters)}\
 ${initialize_output_params(output_parameters)}\
-% if function_name == config['close_function']:
-      session_repository_->remove_session(${service_helpers.create_args(parameters)});
+% if function_name == config['close_function'] or service_helpers.is_custom_close_method(function_data):
```

Comment:
It wouldn't let me do a multi-line suggestion including line 74 but I think we can change 68-74 from 
```
% if function_name == config['close_function'] or service_helpers.is_custom_close_method(function_data):
      session_repository_->remove_session(${service_helpers.create_args(parameters[:1])});
      library_->${function_name}(${service_helpers.create_args(parameters)});
% else:
      auto status = library_->${function_name}(${service_helpers.create_args(parameters)});
      response->set_status(status);
% endif
```
to
```
% if function_name == config['close_function'] or service_helpers.is_custom_close_method(function_data):
      session_repository_->remove_session(${service_helpers.create_args(parameters[:1])});
% endif
      auto status = library_->${function_name}(${service_helpers.create_args(parameters)});
      response->set_status(status);
```
I think we should be setting the response status for these close methods as well.

Reply:
@reckenro I had tried that earlier. But there were system tests for checking that if invlid session is passed to close api, then it won't give error. So it was purposefully left out to not set status for close APIs. @jonlugoatx do you know why was that done?

Reply:
@reckenro , @jonlugoatx , I have made the suggested change and also updated tests. And finally ran the clang-format on the files. Since it was a lot of changes, I have reset your approval votes. Can you please re-review the changes?